builder/qemu/builder.go

```diff
@@ -22,10 +21,10 @@ import (
 const BuilderId = "transcend.qemu"
 
 var accels = map[string]struct{}{
-	"none": struct{}{},
-	"kvm":  struct{}{},
-	"tcg":  struct{}{},
-	"xen":  struct{}{},
+	"none": {},
+	"kvm":  {},
+	"tcg":  {},
+	"xen":  {},
```

Comment:
I think this breaks BC with go 1.4. I'm not sure if we've already done that elsewhere but we should probably make a note in the readme somewhere. You can also add `struct{}` back in to make it BC.